src/work_unit.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Describes a single WorkUnit.
+ */
+
+
+// A work unit is a function that works on the worker thread, and posts a light
+// weight callback to be run on the parent thread.
+WorkUnit = function(fn, on_done, worker) {
+  this.work_fn_ = fn;
```

Comment:
I assume, `work_fn_` is intended to be a private variable. The convention generally is `_workFn` (camel case variablename with a leading _ to denote private)